### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Follow-up to #2563

Question: is `configs/sim/woodpecker/woodpecker_/images/QTvcp Widgets.html` OK to edit or is it generated ? Same question for `share/qtvcp/screens/woodpecker/images/QTvcp Widgets.html` ?

### CODE REVIEW

**Comment by petterreinholdtsen** [comment]
[luzpaz]
> Follow-up to #2563
>
> Question: is `configs/sim/woodpecker/woodpecker_/images/QTvcp
> Widgets.html` OK to edit or is it generated ? Same question for
> `share/qtvcp/screens/woodpecker/images/QTvcp Widgets.html` ?

Both include <meta name="generator" content="AsciiDoc 8.6.7">, making me
suspect they are generated.  Searching for phrases I find
docs/src/gui/qtvcp-widgets.adoc.  Perhaps it is the originating source
file?  Comparing the two I suspect they are different generations of the
same content.

After checking git history of the adoc file, which changed name three
times since 2020, I am convinced the .html files are created using the
adoc file, and should probably be regenerated.
-- 
Happy hacking
Petter Reinholdtsen

**Comment by luzpaz** [comment]
Darn, I made the PR against master again :disappointed:  
lemme see if I can fix that.

**Comment by hansu** [comment]
> Darn, I made the PR against master again disappointed lemme see if I can fix that.

1. grafik

2. grafik

And then rebase & force push your branch (or rebase first)

File: src/emc/usr_intf/gmoccapy/gmoccapy.glade [hunk 1]
```diff
@@ -6190,7 +6190,7 @@ F12 or $ key does the same</property>
               <object class="GtkLabel" id="lbl_ntb_button_manuel">
                 <property name="visible">True</property>
                 <property name="can-focus">False</property>
-                <property name="label">manuel</property>
+                <property name="label">manual</property>
```
Comment (luzpaz) [inline src/emc/usr_intf/gmoccapy/gmoccapy.glade:6193]: please review

**Comment by c-morley** [comment]
These were passed to me from a third party. they are not generated in the linuxcnc project.

**Comment by luzpaz** [comment]
Is there an upstream repo I can send these fixes to, in that case ?

File: src/emc/usr_intf/gmoccapy/gmoccapy.glade (hunk 1 above)
Comment (petterreinholdtsen) [reply src/emc/usr_intf/gmoccapy/gmoccapy.glade:6193]: I would suspect lbl_ntb_button_manuel need to change too?  I can not find any traces of any use of the property or the label, so I guess it is safe to rename only here.

**Comment by petterreinholdtsen** [comment]
I had a closer look at these changes, and all of them seem correct to me.

File: src/emc/usr_intf/gmoccapy/gmoccapy.glade (hunk 1 above)
Comment (luzpaz) [reply src/emc/usr_intf/gmoccapy/gmoccapy.glade:6193]: I'm having issues pushing to my branch. @petterreinholdtsen feel free to push the changes and merge if you deem it worthy ?

**Comment by smoe** [comment]
Seems like something broke the automated tests for 2.9 - and it is not this patch. Any idea anyone?

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> Seems like something broke the automated tests for 2.9 - and it is not
> this patch. Any idea anyone?

This is rather strange.  The rip-and-test action seem to fail in the
tests/raster/ check, which I only find in master, not in 2.9.  Anyone
know what is going on here?

-- 
Happy hacking
Petter Reinholdtsen

**Comment by hansu** [comment]
@luzpaz You can try if a rebase and force push resolves this.

**Comment by smoe** [comment]
@hansu - I think this is fine to be merged. That part that was cancelled is somewhere between buster (working) and bookworm (working).

**Comment by hansu** [comment]
I think a rebase wouldn't hurt anyway ...

**Comment by luzpaz** [comment]
Done.

**Comment by hansu** [comment]
Thanks

**Pull request merged**